### PR TITLE
Service-portal: Add basehref to build for service portal

### PR DESCRIPTION
**Pull request by Thorrik58**
# \<Description\>

The service portal is moving to a new domain using the island.is domain. island.is/minarsidur  will be the base url for My sites
beta.minarsidur.island.is -> island.is/minarsidur 

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by obmagnusson — DISMISSED** [review]
LGTM

File: scripts/ci/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ ARG APP
 ENV APP=${APP}
 ENV NODE_ENV=production
 
-RUN yarn run build ${APP} --prod
+RUN yarn nx run ${APP}:build:production
```
Comment (pshomov) [inline scripts/ci/Dockerfile:24]: I think this is identical change

**Review by dabbeg — COMMENTED** [review]
Have you tested these docker and nginx changes?

File: scripts/ci/Dockerfile (hunk 1 above)
Comment (Thorrik58) [reply scripts/ci/Dockerfile:24]: They are, this shouldve been removed. Ill fix it.

Comment (Thorrik58) [inline scripts/ci/Dockerfile:24]:
```suggestion
RUN yarn run build ${APP} --prod
```

**Comment by obmagnusson** [comment]
> Have you tested these docker and nginx changes?
This pr has a feature deployment 
https://service-portalbase-url-changes-beta.dev01.devland.is/minarsidur and https://service-portalbase-url-changes-beta.dev01.devland.is/  where the serivice-portal and the web pods seem to work just fine seperately. @sindrig can maybe confirm this.

File: apps/service-portal-e2e/src/integration/app.spec.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 describe('service-portal', () => {
   it('should navigate the homepage', () => {
     cy.visit('/')
-    cy.contains('Eldri útgáfa').click()
+    // cy.contains('Eldri útgáfa').click()
```
Comment (sindrig) [inline apps/service-portal-e2e/src/integration/app.spec.ts:4]: is there something else on the page we can check?

**Review by obmagnusson — APPROVED** [review]
LGTM

**Review by dabbeg — COMMENTED** [review]
Everything looks good but I have one problem that I am wondering if it is connected to these changes. When I go to https://service-portalbase-url-changes-beta.dev01.devland.is/minarsidur and log in I get redirected to https://service-portalbase-url-changes-beta.dev01.devland.is/minarsidur/minarsidur/ which shows me "Síða finnst ekki"

**Comment by obmagnusson** [comment]
> Everything looks good but I have one problem that I am wondering if it is connected to these changes. When I go to https://service-portalbase-url-changes-beta.dev01.devland.is/minarsidur and log in I get redirected to https://service-portalbase-url-changes-beta.dev01.devland.is/minarsidur/minarsidur/ which shows me "Síða finnst ekki"

Yes we know of this. As we where not able to replicate this locally with these settings we were hoping that a new deployment on dev would solve this as mabye this is a result of past testing and deployment on this branch?(just a theory) and work from there if the same issue arises on dev. 
The client seems to redirect from the IDS server correctly with just minarsidur/oidc-signin?xxxx url but then decides to redirect from there to /minarsidur/minarsidur which is bad.

**Review by dabbeg — APPROVED** [review]

**Pull request merged**